### DISCUSSION TITLE
Parameterized repo/link for custom lib builder artifacts (custom sdkconfig)

### DISCUSSION THREAD

**Discussion by cziter15** [espressif/arduino-esp32#7864] [Ideas]
I have an idea for the functionality of downloading your own arduino-lib-builder artifacts.

Today I forked the repository, changed the sdkconfig parameters and ran the build actions. After building, I had to copy the libraries manually - and this can be automated.

Since I'm using platformio, it would be nice to have the option to provide an external link to the artifact repository (via platformio.ini), so that each project could have its own sdkconfig settings.

**Comment by me-no-dev** [comment]
the lib-builder has a script to copy the files to a location where the receiving core is: https://github.com/espressif/esp32-arduino-lib-builder/blob/master/tools/copy-to-arduino.sh

**Reply by me-no-dev** [reply]
We are planning on making the libs external, but they still rely on some "internal" configs, like `platform.txt` for Arduino IDE and PIO's python files. They are specific to each lib-builder build.

**Comment by Jason2866** [comment]
@cziter15 haved forked the lib builder too. Modified that a complete framework is builded and released via Github Actions. The framework can directly used in platformio with `platform_packages = framework-arduinoespressif32 @ https://github.com/<owner>/esp32-arduino-lib-builder/releases/download/.....zip`
See https://github.com/Jason2866/esp32-arduino-lib-builder
